docs/program/extend/modular-resources/examples/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Examples of configuring modular resources"
+linkTitle: "Examples"
+description: "Tutorials and repositories with examples of configuring modular resources."
+weight: 10
+type: "docs"
+no_list: true
+image: "/tutorials/img/intermode/rover_outside.png"
+imageAlt: "A large black intermode rover controlled with CANBUS protocol chilling outside in the snow."
+tags: ["server", "rdk", "extending viam", "modular resources", "components", "services"]
+---
+
+The easiest way to familiarize yourself with creating and using modular resources is to:
+
+1. Follow one of these example [tutorials](#tutorials) or clone one of these example [repositories](#repositories).
+2. Control the custom resource on [the Viam app's](https://app.viam.com) [__CONTROL__ tab](/manage/fleet/#remote-control).
+3. Control the custom resource programmatically with a [Viam SDK](/program/sdk-as-client/).
+4. Experiment by changing the resource's behavior.
```

Comment:
And I think these three steps actually aren't helpful as such. They are more what you can do once you've configured it. 

So I'd suggest changing all of this to:

```
To familiarize yourself with creating and using modular resources, follow one of these example [tutorials](#tutorials) or clone one of these example [repositories](#repositories).
Once you have created a modular resource, you can test your custom resource using the [Control tab](/manage/fleet/#remote-control) and program it using the [Viam SDKs](/program/sdk-as-client/).
```

Steps 2-4 don't have any additional help for familiarizing - they're just actually using it, so that's why I think that's a bit weird.